astropy/table/table.py

```diff
@@ -3948,6 +3948,13 @@ class QTable(Table):
 
     """
 
+    def __init__(self, data=None, masked=False, names=None, dtype=None, **kwargs):
```

Comment:
Just a quick look through the PR: here, I think one could write `quantity_dtype=np.inexact` - and then allow the option `None`.
We cannot use `dtype` since that is supposed to give the separate dtype for every column.